docs/github/ios.md

```diff
@@ -0,0 +1,261 @@
+  lane :beta do
+    build
+    api_key = app_store_connect_api_key(
+      key_id: "#{ENV[APPSTORE_KEY_ID]}",
+      issuer_id: "#{ENV[APPSTORE_ISSUER_ID]}",
+      key_filepath: "#{ENV['APPSTORE_P8_PATH']}",
+      duration: 1200, # optional
+      in_house: false, # true for enterprice and false for individual accounts
+    )
+    upload_to_testflight(skip_waiting_for_build_processing:true,api_key: api_key)
+  end
+
+  desc "Create .ipa"
+  lane :build do
+    disable_automatic_code_signing(path: "#{ENV['IOS_BUILD_PATH']}/iOS/Unity-iPhone.xcodeproj")
+    certificates
+    update_project_provisioning(
+      xcodeproj: "#{ENV['IOS_BUILD_PATH']}/iOS/Unity-iPhone.xcodeproj",
+      target_filter: "Unity-iPhone",
+      profile: ENV["sigh_com.company.application_appstore_profile-path"],
+      code_signing_identity: "Apple Distribution: #{ENV['APPLE_TEAM_NAME']} (#{ENV['APPLE_TEAM_ID']})"
+    )
+	increment_build_number(
+       xcodeproj: "#{ENV['IOS_BUILD_PATH']}/iOS/Unity-iPhone.xcodeproj"
+    )
+    gym(
+      project: "#{ENV['IOS_BUILD_PATH']}/iOS/Unity-iPhone.xcodeproj",
+      scheme: "Unity-iPhone",
+      clean: true,
+      skip_profile_detection: true,
+      codesigning_identity: "Apple Distribution: #{ENV['APPLE_TEAM_NAME']} (#{ENV['APPLE_TEAM_ID']})",
+      export_method: "app-store",
+      export_options: {
+        method: "app-store",
+        provisioningProfiles: {
+          ENV["IOS_APP_ID"] => "match AppStore #{ENV['IOS_APP_ID']}"
+        }
+      }
+    )
+  end
+
+  desc "Synchronize certificates"
+  lane :certificates do
+    cleanup_keychain
+    create_keychain(
+      name: keychain_name,
+      password: keychain_password,
+      default_keychain: true,
+      lock_when_sleeps: true,
+      timeout: 3600,
+      unlock: true
+    )
+    match(
+      type: "appstore",
+      readonly: true,
+      keychain_name: keychain_name,
+      keychain_password: keychain_password
+    )
+  end
+
+  lane :cleanup_keychain do
+    if File.exist?(File.expand_path("~/Library/Keychains/#{keychain_name}-db"))
+      delete_keychain(name: keychain_name)
+    end
+  end
+
+  after_all do
+    if File.exist?(File.expand_path("~/Library/Keychains/#{keychain_name}-db"))
+      delete_keychain(name: keychain_name)
+    end
+  end
+
+end
+
+```
+
+### Add Github action
+
+```yaml
+# .github/workflows/main.yml
+
+buildForiOsPlatform:
+  name: Build for ${{ matrix.targetPlatform }} on version ${{ matrix.unityVersion }}
+  runs-on: ubuntu-latest
+  strategy:
+    fail-fast: false
+    matrix:
+      projectPath:
+        - .
+      unityVersion:
+        - 2020.2.2f1
+      targetPlatform:
+        - iOS
+  steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: Library
+        key: Library-${{ matrix.targetPlatform }}-v2
+    - uses: webbertakken/unity-builder@v2.0-alpha-6
+      with:
+        projectPath: ${{ matrix.projectPath }}
+        unityVersion: ${{ matrix.unityVersion }}
```

Comment:
It's better to leave out unityVersion, since the Unity Actions will now default to getting the correct version from ProjectSettings/ProjectVersion.txt